man/bootctl.xml

```diff
@@ -153,6 +153,21 @@
         disables the timeout while always showing the menu. When an empty string ("") is specified the
         bootloader will revert to its default menu timeout.</para></listitem>
       </varlistentry>
+
+      <varlistentry>
+        <term><option>unlink</option> <replaceable>ID</replaceable></term>
```

Comment:
please add all these new options/verbs to the shell completion scripts

Reply:
*sigh* how many slices does the salami have?

Reply:
hopefully enough to keep everyone well fed!